common/app/views/fragments/items/elements/facia_cards/kicker.scala.html

```diff
@@ -7,7 +7,8 @@
    elsewhere). Given that, how much of the below do we need? *@
 
 @makeLink(uri: String, inner: String) = {
-    <a href="@LinkTo(uri)" data-link-name="kicker" class="fc-item__kicker @classes.mkString(" ")">@Html(inner)</a>
+    @* Support nested links by wrapping inner links in <object> element *@
+    <object type="nested/link" class="u-nested-link"><a href="@LinkTo(uri)" data-link-name="kicker" class="fc-item__kicker @classes.mkString(" ")">@Html(inner)</a></object>
```

Comment:
i wonder if we should extract these into a `nestedLink` helper, and add the IE conditional comments? have a feeling it will totally bork < IE9, and it should at least _work_ there, even it not well?

Reply:
helper may be overkill though

Reply:
Yeah, IE9 looks terrible even now... the CSS doesn't seem to load, I'm guess due to some JS compatibility issue. Not sure we should even factor in IE9 support these days, the usage is around 0.27% (incidentally higher than both IE9 and IE10). This can be a question for another time.

I'll add the conditional comments that will kill sublinks on IE9. I think extracting into a helper might be an exercise of DRYness gone too far, as we'd have to find a way of passing all possible attributes to the nested `<a>` or `<button>`. I'm not sure the eventual helper function would be any easier to use than copy-pasting an existing nested link?

If we could do something like:

```js
// helper.js

export const NestedLink = ({ children }) => (
  <object type="nested/link">
    {children}
  </object>
)
```

...then we'd be in business!